### PR TITLE
attester: docs:  running the fobtain-attester

### PR DESCRIPTION
**Pull request by 0xDen**
Signed-off-by: Denis <denis.semakin@3mdeb.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +65,26 @@ cd tpm2-tools
...
+additionally configured this library performs following steps:
+
+1. It tries to open corresponding device files "/dev/tpmrm0" or "/dev/tpm0".
+   In this case operation requires CAP_SYS_ADMIN capability for process.
+   In other words it must be run with superuser rights, e.g.:
+```shell
+$ sudo ./bin/fobnail-attester
+```
+
+2. If the first step got faied then the the application creates a TCP socket and
```
Comment (arturkow2000) [inline README.md]:
```suggestion
2. If the first step failed then the the application creates a TCP socket and
```

**Review by arturkow2000 — APPROVED** [review]
@0xDen One suggestion, rest looks good.

File: README.md (hunk 1 above)
Comment (0xDen) [reply README.md]: Fixed

**Comment by 0xDen** [comment]
Merged

**Pull request merged**